trade-finance-manager-api/src/v1/__mocks__/api.js

```diff
@@ -82,7 +82,9 @@ module.exports = {
     };
     return Promise.resolve(updatedFacility);
   },
-  updateGefFacility: (facilityId, facilityUpdate) => Promise.resolve(facilityUpdate),
+  updateGefFacility: ({ facilityUpdate }) => {
+    return facilityUpdate;
```

Comment:
Should this be async? A badly written function that didn't await `updateGefFacility` could pass a test with this mock right?

Reply:
uopdated